lib/showcase_generator.dart

```diff
@@ -86,6 +83,10 @@ See https://github.com/flutter/flutter-intellij/wiki/Using-live-preview
     final double boundaryHeight =
         annotatedElement.annotation.peek('height').doubleValue;
 
+    final showcase = forDesignTime.returnType.name.contains("List")
```

Comment:
There's probably a way to check this type consistently (and guarantee it's a List<Widget>),

Also, `forDesignTime` can be null, as it's not required for a Widget to implement it.

Reply:
There's another problem here: `forDesignTime` is a class factory, so it's not possible for it to return a List, it must return an instance of the widget it's declared. I've tried to create mixins for it, but you can't have mixins for static methods, nor factories.